recipes/pyomero-upload/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
this could be moved into the meta.yml if you like

Reply:
Thanks, I did not know that was possible but I see the meta.yml option in the docs now. Is it okay if we proceed with the build.sh script since it may be necessary to add more to it in further releases?